libs/domain/picking/user-profile/user-profile.component.ts

```diff
@@ -107,6 +110,7 @@ export class UserProfileComponent extends LitElement {
   }
 
   protected onLogOut(): void {
+    this.logoutLoading = true;
     this.authService.logout();
```

Comment:
Is it possible to get error on logout func?
Should we change `logoutLoading` to false then?